features/step_definitions/advocate_new_claim_steps.rb

```diff
@@ -411,6 +411,32 @@
   create(:fee, :misc,  claim: claim, quantity: 2, amount: 5.0)
 end
 
+Given(/^I am on the new claim page with Daily Attendance Fees in place$/) do
+  create(:fee_type, :basic, description: 'Basic Fee', code: 'BAF')
+  create(:fee_type, :basic, description: 'Daily attendance fee (3 to 40)', code: 'DAF')
+  create(:fee_type, :basic, description: 'Daily attendance fee (41 to 50)', code: 'DAH')
+  create(:fee_type, :basic, description: 'Daily attendance fee (51+)', code: 'DAJ')
+  visit new_advocates_claim_path
+end
+
+When(/^I fill in the trial details$/) do
+  select2('Trial', from: 'claim_case_type_id')
+  fill_in 'claim_actual_trial_length', with: 1
```

Comment:
I dont think this is required because the actual trial length is set to 1 only to then be changed on this line https://github.com/ministryofjustice/advocate-defence-payments/pull/841/files#diff-0ed77b0f3d6ca4f188322cbe3a22a955R191 in the feature. Maybe this step could be renamed to something like 'I select the case type as a trial'  (on a side note perhaps this should really be a global/shared step to set the case type as I'm sure other features use/ would do something similar)